images/bootstrap/create_bazel_cache_rcs.sh

```diff
@@ -24,9 +24,9 @@ package_to_version () {
 # look up a binary with which and return the debian package it belongs to
 command_to_package () {
     local binary_path
-    binary_path=$(readlink -f "$(which "$1")")
-    # `dpkg -S $package` spits out lines with the format: "package: file"
-    dpkg -S "$1" | grep "${binary_path}" | cut -d':' -f1
+    binary_path=$(readlink -f "$(command -v "$1")")
+    # `dpkg -S $file` spits out lines with the format: "package: file"
```

Comment:
you're not really just passing "$file" here, you're passing the fully resolved and de-symlinked path to the file. May be worth clarifying (this seems to work on my debian box, but don't know all the corner cases of dpkg).

Reply:
It seems to work with the unresolved symlink as well, but will potentially output more than one match in that case.

Reply:
I've switched to dpkg-query --search for clarity (which dpkg -S forwards to), and updated the comments